binding/lua/multiverso-scm-1.rockspec

```diff
@@ -0,0 +1,35 @@
+package = "multiverso"
+version = "scm-1"
+
+source = {
+    url = "https://github.com/Microsoft/multiverso"
+}
+
+description = {
+    summary = "Torch binding for multiverso.",
+    detailed = [[
+        Multiverso is a parameter server framework for distributed machine
+        learning, this package make it possible to have multiverso embedded in
+        your original torch programs.
+    ]],
+    homepage = "http://www.dmtk.io",
+    license = "MIT"
+}
+
+dependencies = {
+    "torch >= 7.0"
+}
+
+build = {
```

Comment:
Currently, the build is supposed to run as the folder where it exists. Same behaviour as having it as a standalone torch package.